src/SIPdump.h

```diff
@@ -361,8 +361,9 @@ int find_value(const char *value, const char *buffer, char *outbuf,
 		}
 	}
 
-	strncpy(outbuf, tempbuf, outbuf_len);
-	outbuf[outbuf_len] = 0;
+	strncpy(outbuf, tempbuf, outbuf_len - 1);
+	if (outbuf_len > 0)
+		outbuf[outbuf_len - 1] = 0;
```

Comment:
I have now looked at all the code paths leading to this code (i.e. `if (buflen > 0)` check), and it seems that `outbuf_len` can't be zero. I will remove this weird check.